site/src/atomics/width.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Width Atomics
+description: Width related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Width Atomics
+
+Need to specifiy a specific width for an element? You've come to the right place! These classes should be used sparingly, and only used after testing heavily at all screen widths (starting with mobile). That said, if you need to specify and min, max or normal width, they can be a useful tool in the utility belt.
+
+Values are in pixels. Keep in mind that the `min-width` and `max-width` properties override the standard `width` property. If you aren't using the Atlas core folder as a base for you styles, these classes work best when [you're setting `box-sizing` to be `border-box` in some way](https://css-tricks.com/box-sizing/#aa-present-day-box-sizing).
+
+| cssproperty                       | value (px)                                                    | screensize                        |
+| --------------------------------- | ------------------------------------------------------------- | --------------------------------- |
+| `width`, `max-width`, `min-width` | `100`, `150`, `200`, `250`, `300`, `350`, `400`, `450`, `500` | `tablet`, `desktop`, `widescreen` |
+
+## Usage
+
+Set the `width`, `min-width`, or `max-width` of an element. of an element
+
+```html
+<div class="width-100 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>100</p>
+</div>
+<div class="width-150 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>150</p>
+</div>
+<div class="width-200 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>200</p>
+</div>
+<div class="width-250 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>250</p>
+</div>
+<div class="width-300 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>300</p>
+</div>
+<div class="width-350 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>350</p>
+</div>
+<div class="width-400 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>400</p>
+</div>
+<div class="width-450 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>450</p>
+</div>
+<div class="width-500 box-shadow-heavy padding-sm font-weight-semibold">
+	<p>500</p>
+</div>
+```
+
+### Responsive rules
+
+If there are screen-specific class variations available they can be described in this section.
+
+```html
+<div class="width-200 width-400-tablet box-shadow-heavy padding-sm font-weight-semibold">
+	<p>300 ➡ 500</p>
```

Comment:
Per applied classes it should be `200 ➡ 400`

Reply:
🤦‍♂️ thanks!